apps/antalmanac/src/components/RightPane/SectionTable/CourseInfo/CourseInfoSearchButton.tsx

```diff
@@ -20,7 +21,6 @@ export function CourseInfoSearchButton({ courseDetails, term }: { courseDetails:
         RightPaneStore.updateFormValue('deptValue', deptCode);
         RightPaneStore.updateFormValue('courseNumber', courseNumber);
         RightPaneStore.updateFormValue('term', term);
-
         displaySections();
         setActiveTab(1);
     }, []);
```

Comment:
```suggestion
    }, [courseNumber, deptCode, displaySections, setActiveTab, term]);
```

issue: this callback has incorrect dependency array. (I accidentally introduced this while my eslint was fritzing)